ci/azure-pipelines-release.yml

```diff
@@ -0,0 +1,91 @@
+# Copyright the Hyperledger Fabric cont ributors. All rights reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+name: RELEASE-$(Date:yyyyMMdd)$(Rev:.rrr)
+trigger: none
+pr: none
+
+variables:
+  - group: credentials
+  - name: GOPATH
+    value: $(Agent.BuildDirectory)/go
+  - name: GOVER
+    value: 1.12.12
+  - name: RELEASE
+    value: 1.4.4
```

Comment:
This seems weird to me. This value should be explicitly populated by the person doing the manual execution of the build. According to the doc, having this in the pipeline definition means it can't be changed:

> If a variable appears in the variables block of a YAML file, it is fixed and cannot be overridden at queue time.

Reply:
Removed, I was aware of this, and had good intentions behind it (I originally had a comment as to how this would be addressed in the future), but I guess I don't really have an issue with this,